site/content/docs/0.0/migration-from-boosted.md

```diff
@@ -136,6 +143,73 @@ Technically, it means that you can get rid of the following things:
       <li><code>$ouds-border-width-thick</code></li>
       <li><code>$ouds-border-width-thicker</code></li>
       <li><code>$ouds-border-width-thin</code></li>
+      <li><code>$ouds-elevation-blur-0</code></li>
+      <li><code>$ouds-elevation-blur-100</code></li>
```

Comment:
I haven't checked in other lists, but do we add this Sass variable in this list even if it is commented and not used? In the end, this is not really usable.

Reply:
Resolved by https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2741/commits/775596980bbcfe1e39cf500d1f539a09984f027c

Reply:
Just checked that point and you're right we did not mentioned commented variables. This make sens.
I pushed a fix on this point.